index.html

```diff
@@ -205,6 +205,23 @@ <h1 class="about">About Myself</h1>
         </div>
     </section>
 
-    
+    <section class="contact-form">
+        <div class="form-1">
+
+            <h1 class="Contact">Contact me</h1>
+            <p class="application">If you have an application you are interested 
+                in developing, a feature that you need built or a project that needs coding. 
+                I'd love to help with it </p> 
+
+            <form action="https://formspree.io/f/xrgvkako" method="post">
+                <input name="name" id="name" type="text" placeholder="Yeremias" maxlength="30" required>
+                <input name="email" id="email" type="email" placeholder="natajayan14@gmail.com" required>
+                <textarea name="textarea" id="textarea"textarea cols="30" rows="10" placeholder="Write your message here" maxlength="500" required></textarea>
```

Comment:
- [Optional] I would like to suggest that you add padding to the `input` and `textarea`  form elements to give a good user experience when someone is filling in the form. When I fill in the form the text touches the edges of the input elements and this can be improved by adding padding to the `input` and `textarea` elements. Please see the screenshot below for more details on this.

```CSS
* {
  margin: 0;
  padding: 0;
  box-sizing: border-box;
}

form input {
  padding-left: 20px;
  margin-top: 1.25rem;
}

textarea {
  padding-left: 20px;
  background: #fff;
  border: 1px solid #cfd8dc;
  font-size: 15px;
  padding-top: 10px;
  border-radius: 8px;
  width: 327px;
  height: 170px;
  margin-bottom: 20px;
  margin-top: 20px;
}


```

## Form elements without padding
<img src="https://user-images.githubusercontent.com/16392046/222588416-677c22fa-ea1d-4f30-9463-c22d460385e0.png" width="375px" />

## Form elements with padding 
<img src="https://user-images.githubusercontent.com/16392046/222589446-55a66534-28b0-44d8-b582-2de5bbbba247.png" width="375px"  />